ext/B/B.xs

```diff
@@ -1393,6 +1393,20 @@ aux_list(o, cv)
                 XSRETURN(len);
 
             } /* OP_MULTIDEREF */
+
+        case OP_MULTIPARAM:
+            {
+                struct op_multiparam_aux *p = (struct op_multiparam_aux *)aux;
+                UV nparams = p->n_positional;
+                EXTEND(SP, (IV)(3 + nparams + 1));
+                mPUSHu(p->min_args);
+                mPUSHu(p->n_positional);
+                PUSHs(sv_2mortal(p->slurpy ? newSVpvf("%c", p->slurpy) : &PL_sv_no));
```

Comment:
Why run a 1 byte long string through a printf engine?

Reply:
Because offhand I don't believe we have a `newSVpvc` function. Can you suggest an alternative?

Reply:
`newSVpvn(&p->slurpy, 1)`?

Also, why are we mortalizing `PL_sv_no`? Couldn't we do this (warning: untested code, written in the browser):
```c
PUSHs(p->slurpy ? newSVpvn_flags(&p->slurpy, 1, SVs_TEMP) : &PL_sv_no);
```